tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -79,6 +79,14 @@ stages:
 
     - template: "templates/use-android-ndk.yml"
 
+    - script: |
```

Comment:
these "Check Android NDKs" steps don't seem to show anything. e.g.:
https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=1431601&view=logs&j=b27c03c3-79f9-5527-4b3a-dcfd5ff9627d&t=4c403f51-f20d-5048-3dee-14ef4d2220b5

might be because of the line continuations `\` that join all the lines into the first set command.

if these steps are not showing anything now, consider removing them.

Reply:
It will also be removed from PR https://github.com/microsoft/onnxruntime/pull/21340